tests/spec/helpers/realm-logging.js

```diff
@@ -27,11 +27,11 @@ function patch(Realm) {
         const originalClearTestState = Realm.clearTestState;
         Realm.clearTestState = function patchedClearTestState() {
             originalClearTestState.call(Realm);
-            setupLogging(Realm);
+            //setupLogging(Realm);
         }
 
         // Setup the logger initially
-        setupLogging(Realm);
+        //setupLogging(Realm);
```

Comment:
if not needed please remove the commented code

Reply:
I will add a FIXME; we need to figure out a way to enable general logging.